doc/api/next_api_changes/2019-04-21-AL.rst

```diff
@@ -0,0 +1,7 @@
+Removals
+````````
+
+The following API elements have been removed:
+- ``backend_gtk3.PIXELS_PER_INCH``,
+- ``backend_pgf.re_escapetext``, ``backend_pgf.repl_escapetext``,
+  ``backend_pgf.re_mathdefault``, ``backend_pgf.repl_mathdefault``,
```

Comment:
/home/circleci/project/doc/api/next_api_changes/2019-04-21-AL.rst:7: WARNING: Unexpected indentation.

likely causes the doc build failure.

Reply:
thanks, should be fixed now